src/docs/sdk/event-payloads/contexts.mdx

```diff
@@ -147,7 +147,7 @@ The `type` and default key is `"os"`.
 
 `name`
 
-: **Required**. The name of the operating system.
+: _Optional_. The name of the operating system. It might be derived from `raw_description`. It is **required** if `raw_description` is not provided.
```

Comment:
These should at least be recommended. Even with `raw_description`, SDKs are encouraged to try obtain better values.

```suggestion
: _Recommended_. The name of the operating system. It might be derived from `raw_description`. It is **required** if `raw_description` is not provided.
```

Reply:
That's fair, updated both.